### PR TITLE
add doc for F.softplus

### PR DESCRIPTION
**Pull request by ssnl**

### CODE REVIEW

**Comment by vadimkantorov** [comment]
Seems this adds docs for F.softplus (not F.softmax). And F.softplus is `log(1 + exp(x))` (https://pytorch.org/docs/stable/nn.html?highlight=softplus#torch.nn.Softplus), not the formula you mention in the patch, right? (softplus isn't normalized over any other values)

**Comment by SsnL** [comment]
lol @vadimkantorov  my bad. I will update this to add doc for softplus

File: torch/nn/functional.py [hunk 1]
```diff
@@ -1160,6 +1160,10 @@ def softsign(input):
 
 softplus = _add_docstr(torch._C._nn.softplus, r"""
 softplus(input, beta=1, threshold=20) -> Tensor
+
+Applies element-wise, the function :math:`\text{Softplus}(x) = \frac{1}{\beta} * \log(1 + \exp(\beta * x))`
```
Comment (zou3519) [inline torch/nn/functional.py]: Should we mention `threshold` at all? It's not 100% true that we are applying this function; Softplus applies this function  until the values exceed the threshold

Comment (SsnL) [reply torch/nn/functional.py]: Good point! removed!

Comment (vadimkantorov) [reply torch/nn/functional.py]: this argument is not supported in F.softplus? or is it just not intended for public use?

Comment (zou3519) [reply torch/nn/functional.py]: Sorry, I was suggesting that we add `threshold` to the formula or write a line about it explaining how `\text{Softplus}(x) = \frac{1}{\beta} * \log(1 + \exp(\beta * x))` is only true until `threshold`

Comment (SsnL) [reply torch/nn/functional.py]: oh i see. sorry. i'll update.

Comment (SsnL) [reply torch/nn/functional.py]: Updated!

**Review by zou3519 — APPROVED** [review]
